doc/user-manual/language/reflection.lagda.rst

```diff
@@ -711,3 +711,32 @@ Example usage:
       defId id-name
 
     unquoteDecl id′ = mkId id′
+
+System Calls
+~~~~~~~~~~~~
+
+It is possible to run system calls as part of a metaprogram, using the ``execTC`` primitive. You can use this feature to implement type providers, or to call external solvers. For instance, the following example calls ``/bin/echo`` from Agda:
+
+.. code-block:: agda
+
+  postulate
+    execTC : String → List String → String
```

Comment:
It's quite clear from the example, but for absolute clarity I think it would be good to describe/name the inputs and outputs to `execTC`.

Reply:
I've added a paragraph explicitly documenting the arguments and the output, and I've changed the postulate to have named arguments.